src/fklearn/validation/validator.py

```diff
@@ -222,6 +243,9 @@ def parallel_validator(train_data: pd.DataFrame,
     predict_oof : bool
         Whether to return out of fold predictions on the logs
 
+    return_train_score : bool
```

Comment:
This is because the **current** behavior differs for both functions. For some reason with current behavior, `parallel_validator` always returns only the first train log while `parallel_validator` always returns all train logs. I feel they could be made consistent but in this PR I opted to preserve this behavior for both (when using default parameters).

Reply:
got it